FrontEnd/src/components/ProfileDetail/MainInfo/ProfileDetailNavBar.jsx

```diff
@@ -20,6 +32,16 @@ function ProfileDetailNavBar({ data }) {
   const { activeLinks } = useContext(ActiveLinksContext);
   const [activeLink, setActiveLink] = useState('');
 
+  const companyType = () => {
```

Comment:
Should we return default links set for unregistered users?

Reply:
Here we return links for both registered and unregistered users so they could view company profile. Tabs correspond to the type of viewed company.